### ISSUE TITLE
Generic OctoPlane is not supported by Control Allocation Geometry neither by UAVCAN

### ISSUE THREAD

**Issue by atikhono1972** [PX4/PX4-Autopilot#19857]
## Describe the bug
A clear and concise description of the bug.

1) Control Allocation Geometry in the release 1.13.0 limits the number of motors to 8.
2) Number of ESCs is limited by UAVCAN to 8

Both 1) and 2) dont allow to configure the Standard VTOL in the Octoplane Configuration with UAVCAN ESCs

## To Reproduce

- Open GUI of Actuators in the QGroundControl while FMU board is configured with Generic Octoplane VTOL 

## Expected behavior

Number of motors shall be at least 9  in the Control Allocation Geometry
Number of the ESCs shall be at least 9 in the UAVCAN actuators

## Log Files and Screenshots
*Always* provide a link to the flight log file:
- Download the flight log file from the vehicle ([tutorial](https://docs.px4.io/master/en/getting_started/flight_reporting.html)).
- Upload the log to the [PX4 Flight Review](http://logs.px4.io/)
- Share the link to the log (Copy and paste the URL of the log)

Add screenshots to help explain your problem.

## Drone (please complete the following information):
- Describe the type of drone.
- Photo of the IMU / autopilot setup if possible.

## Additional context
![2022-06-30](https://user-images.githubusercontent.com/62332250/176731830-ed2ab597-6024-4bb5-8c16-4cbcacc538ed.png)

**Comment by phinchey** [comment]
@bkueng is there any plan to increase the number of actuators in the control allocation to support Octoplanes and others?

**Comment by bkueng** [comment]
Yes we have others asking for it as well. The factors that need to be considered are flash size and uorb message size.

**Comment by phinchey** [comment]
Ok, I'll be patient, thank you for all your hard work!
